bedrock/firefox/templates/firefox/welcome/page15/base.html

```diff
@@ -0,0 +1,202 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/welcome/base.html" %}
+
+{% block page_title %}{% endblock %}
+
+{% block page_image %}{{ static('img/products/vpn/common/social-share.png') }}{% endblock %}
+
+{% block body_class %}welcome-page15{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('firefox_welcome_page15') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-15' %}
+{% set _utm_campaign = 'welcome-15-vpn' %}
+
+{% block experiments %}
+  {% if switch('welcome-15-experiment', ['en-US', 'de', 'fr']) %}
+    {{ js_bundle('firefox_welcome_page15_experiment') }}
+  {% endif %}
+{% endblock %}
+
+{% block shoulder_logo %}
+  {{ picture(
+    url="protocol/img/logos/firefox/browser/logo-word-hor.svg",
+    sources=[
+      {
+        "media": "(prefers-color-scheme: light)",
+        "srcset": {
+          "protocol/img/logos/firefox/browser/logo-word-hor.svg": "default",
+        }
+      },
+      {
+        "media": "(prefers-color-scheme: dark)",
+        "srcset": {
+          "protocol/img/logos/firefox/browser/logo-word-hor-white.svg": "default",
+        }
+      }
+    ],
+    optional_attributes={
+      "class": "c-header-logo",
+      "alt": "Firefox",
+      "width": "216",
+      "height": "40",
+    }
+  ) }}
+{% endblock %}
+
+{% block shoulder_cta %}{% endblock %}
+
+{% block content_intro %}
+    {% if variation == '2' %}
+      <div class="c-main variation-2 mzp-l-content mzp-t-content-lg">
+        <img class="c-main-logo" src="/media/protocol/img/logos/mozilla/vpn/logo-flat.svg" width="60" alt="">
+        <h1 class="c-main-title">{{ var_1_main_title }}</h1>
+        <p class="c-main-tagline">
+          {{ var_1_main_tagline }}
+        </p>
+        {{ vpn_product_referral_link(
+          referral_id='welcome-page15-v2',
+          link_to_pricing_page=True,
+          link_text=cta_text,
+          class_name='mzp-t-product mzp-t-xl',
+          optional_attributes= {
+                'data-cta-text' : 'Try Mozilla VPN',
+                'data-cta-type' : 'button'
+          }
+        )}}
+      </div>
+    {% else %}
+      <div class="c-main variation-1 mzp-l-content mzp-t-content-md">
+        <img class="c-main-logo" src="/media/protocol/img/logos/mozilla/vpn/logo-flat.svg" width="60" alt="">
+        <h1 class="c-main-title">{{ var_2_main_title }}</h1>
```

Comment:
A minor thing but I can anticipate a designer complaining about the text wrap on the v1 headline in en-US and en-CA... maybe this needs a max-width added? v2 has a narrower container already so it's fine.

![image](https://github.com/mozilla/bedrock/assets/205591/13405547-fcd4-44c4-8393-12c730090cf3)